tests/compliance_operator_v2_test.go

```diff
@@ -431,3 +431,30 @@ func TestComplianceV2ScheduleRescan(t *testing.T) {
 	// Assert the scan is rerunning on the cluster using the Compliance Operator CRDs
 	waitForComplianceSuiteToComplete(t, scanConfig.ScanName, 2*time.Second, 5*time.Minute)
 }
+
+func TestComplianceV2CreateScanConfigWithInvalidCluster(t *testing.T) {
+	conn := centralgrpc.GRPCConnectionToCentral(t)
+	client := v2.NewComplianceScanConfigurationServiceClient(conn)
+	clusterId := uuid.NewV4().String()
+
+	scanConfigName := "cis-scan-schedule"
+	sc := v2.ComplianceScanConfiguration{
+		ScanName: scanConfigName,
+		ScanConfig: &v2.BaseComplianceScanConfigurationSettings{
+			OneTimeScan: false,
+			Profiles:    []string{"ocp4-cis", "ocp4-cis-node"},
+			ScanSchedule: &v2.Schedule{
+				IntervalType: 3,
+				Hour:         0,
+				Minute:       0,
+			},
+			Description: "Scan schedule for CIS profiles to run daily.",
+		},
+		Clusters: []string{clusterId},
+	}
+	_, err := client.CreateComplianceScanConfiguration(context.TODO(), &sc)
+	fmt.Println(err)
```

Comment:
Need to clean this up in a subsequent patch, but this can ultimately be used in the assertion.

Reply:
Any reason to do this in the e2e test vs simply doing it in the `manager_impl_test`?  I don't think we should drive everything in the e2e test as there are costs (time and $$) associated with that.

Reply:
Hit a minor issue in the unit tests with error assertions. Attempting to fix in https://github.com/stackrox/stackrox/pull/9942.